src/pages/SettingsPage/UsersSettings/userDetail.jsx

```diff
@@ -280,6 +283,7 @@ const UserDetail = ({
         }).unwrap()
 
         toast.update(toastId.current, { render: `Updated user: ${user.name} ` })
+        user.self && dispatch(updateUserData(data)) && dispatch(updateUserAttribs(data))
```

Comment:
Only `dispatch(updateUserAttribs(data))` gets dispatched as ` user.self && dispatch(updateUserData(data))` is the ternary operator. 

If we had `onMyUserUpdate` then we would only need to dispatch that one action. 

https://redux.js.org/style-guide/#avoid-dispatching-many-actions-sequentially

Reply:
Sorry `dispatch(updateUserData(data))` does get called because it's checking if the value is true or false.

If for any reason `dispatch(updateUserData(data))` was false then `dispatch(updateUserAttribs(data))` would not run. This should be an `if` statement.

Reply:
Also try updating the fullName and pressing save, it doesn't update the fullName in the user menu. Probably because the wrong data is being sent.


https://github.com/ynput/ayon-frontend/assets/49156310/f1f2adcb-c85f-4724-a539-01cd318f0813


![image](https://github.com/ynput/ayon-frontend/assets/49156310/c83e2a88-3580-4a97-8a2c-560a862d3997)